rfcs/components/reducing-tooltips.md

```diff
@@ -0,0 +1,68 @@
+- Try to write a neutral problem statement, not one that motivates your desired solution
+- Remember, "Writing is thinking". It's natural to realize new ideas while writing your proposal
+-->
+
+---
+
+Contributors: @smhigley
+
+## Summary
+
+This RFC proposes removing all built-in tooltips from our default shipped components, with the exception of icon text alternatives. For cases where we choose to include icon tooltips, the proposal is to use a custom tooltip rather than the `title` attribute.
+
+## Background
+
+Tooltips have a long history with overuse and poor accessibility, going back to their origin in early graphical OS's and web browsers. From the start, they were designed to work with a mouse and not much else. The end result is that now we have a bad habit of creating tooltip UI that is inaccessible, to a greater or lesser extent, with keyboard, non-mouse pointer, zoom, magnification, switch, and voice control.
+
+Additionally, not all tooltips are equal: a tooltip on a static text container will never be accessible without a mouse, while a custom tooltip on an input could be made to work with mouse, keyboard, and other pointers. Tooltips that contain supplementary but unnecessary text do not cause as much harm when they cannot be accessed.
+
+Tooltips used for truncation are often the worst of the bunch: they disproportionately impact people relying on zoom or small screens, and those people are also the ones who encounter more accessibility barriers interacting with them. Even without the accessibility barrier of the tooltip itself, truncation + tooltips creates a vastly more labor-intensive and unequal reading/skimming experience for people who need to increase text size. With WCAG 2.1's Reflow and Content on Hover/Focus criteria applying new requirements for zoom accessibility and tooltip accessibility, many of our current truncation tooltips will likely become compliance bugs in the near future as well.
+
+Many of these forms of inaccessibility are not tested for, so we are continuing to harm people without catching it. Some of them are tested for, and we have difficult-to-resolve bugs against Fluent v7 and v8. Those include tooltips used in Persona/FacePile, DatePicker, DetailsList, Nav, picker options, and SwatchColorPicker.
+
+The problems with the `title` attribute are even worse: the tooltip UI shipped by browsers is not keyboard-accessible at all, it cannot be dismissed on demand, it is not persistent, it does not scale with zoom or OS/browser font size settings, and does not respond to other font and color customizations.
+
+There is more detailed information on tooltip accessibility problems and requirements in this [tooltip talk](https://www.youtube.com/watch?v=lb0_v7D4kbs) and [tooltip article](https://sarahmhigley.com/writing/tooltips-in-wcag-21/).
+
+In addition to tooltip accessibility, adding Tooltip as a dependency of other components also increases the bundle size for those components.
```

Comment:
@behowell wouldn't mind your input on how to word this better, and also how to concisely explain the issue with loading internal-dependency tooltips on a delay 😄